lib/auth/in_memory/backend.js

```diff
@@ -3,16 +3,37 @@
 const crypto = require('crypto');
 
 const errors = require('../../errors');
-const accountsKeyedbyAccessKey =
-    require('./vault.json').accountsKeyedbyAccessKey;
-const accountsKeyedbyCanID =
-    require('./vault.json').accountsKeyedbyCanID;
-const accountsKeyedbyEmail =
-    require('./vault.json').accountsKeyedbyEmail;
 const calculateSigningKey = require('./vaultUtilities').calculateSigningKey;
 const hashSignature = require('./vaultUtilities').hashSignature;
+const Indexer = require('./indexer');
+
+function _buildArn(generalResource, specificResource) {
+    return `arn:aws:s3:::${generalResource}/${specificResource}`;
+}
+
+function formatResponse(userInfoToSend) {
+    return {
+        message: {
+            body: { userInfo: userInfoToSend },
+        },
+    };
+}
+
+/**
+ * Class that provides a memory backend for verifying signatures and getting
+ * emails and canonical ids associated with an account.
+ *
+ * @class Backend
+ */
+const backend = class Backend {
```

Comment:
It's unnecessary to create a const as well as define a named class. I recommend 
```js
class Backend {

}
module.exports = Backend;
```
Btw, make sure you name the file Backend.js, as by convention,  the capitalization of filenames  signifies classes.